chat/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "muchat_chat"
```

Comment:
For the name of the crates, use `-` instead of `_`, that's an standard on rust projects.